docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -77,6 +77,20 @@ function App() {
 
 To use the `v5` version of MUI Core, you first need to update the package names:
 
+```sh
+npm install @mui/material @mui/styles 
+
+// or with `yarn`
+yarn add @mui/material @mui/styles
+```
+
+**Optional**: if you have one these packages, install the new package separately
+- For `@material-ui/lab`, install `@mui/lab`
+- For `@material-ui/icons`, install `@mui/icons-material`
+
+<details>
+<summary>See all packages change</summary>
+
```

Comment:
Before the change it is very hard to know what should be the update. I think this is what most developer are looking for, the installation part with `npm install` or `yarn add`.